.github/workflows/pr-checks-workflow-references.yml

```diff
@@ -0,0 +1,67 @@
+# Checks that all references to the 'plugin-ci-workflows' actions are pointing to the 'main' branch in PRs.
+name: Check @main workflow references
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check-references:
+    # Ignore PRs created by release-please. Those PRs will have their references switched to the new version
+    # in order to create hermetic releases, so this action would always fail on those PRs.
+    if: startsWith(github.event.pull_request.head.ref, 'release-please--branches--main--') == false
+    runs-on: ubuntu-arm64-small
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          persist-credentials: false
+
+      - name: Switch references
+        id: switch-references
+        uses: grafana/plugin-ci-workflows/actions/internal/switch-references@main
+        with:
+          repository: grafana/plugin-ci-workflows
+          ref: main
+          paths: |
+            .github/workflows/ci.yml
+            .github/workflows/cd.yml
+            .github/workflows/playwright.yml
+            .github/workflows/playwright-docker.yml
+            actions/plugins/**
+
+      - name: Check drift
+        if: steps.switch-references.outputs.changed == 'true'
+        run: |
+          git diff
```

Comment:
this diff (line 43) is a leftover from testing?

Reply:
I wanted to leave it so the raw git diff is still available in the GitHub actions logs, just in case there are any issues with the awk script below for the fancy GitHub actions annotations